doc/manual/overview.rst

```diff
@@ -139,6 +139,7 @@ Bert Peters,
 Ludo Pulles,
 Tobias Polzer,
 Jeroen Schot,
+Maarten Sijm,
```

Comment:
I did not add this on purpose since he doesn't list his name publicly, but I assume @mpsijm is ok with this?

Reply:
Makes sense, will take it out.

Reply:
It doesn't really matter, as my name is listed as "commit author" anyway :joy: Realising this, I might as well list my name publicly, thanks for pointing that out :slightly_smiling_face:

Reply:
I've added you (back) to the list.